daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md

```diff
@@ -1,9 +1,9 @@
 ---
 type: docs
-title: "How-To: Install Dapr into a Kubernetes cluster"
-linkTitle: "(optional) Init Dapr on Kubernetes"
-weight: 70
-description: "Install Dapr in a Kubernetes cluster"
+title: "Deploy Dapr on a Kubernetes cluster"
+linkTitle: "Deploy Dapr"
+weight: 20000
+description: "Follow these steps to deploy Dapr on Kubernetes."
```

Comment:
We should add an alias (https://gohugo.io/content-management/urls/#aliases) here for the old link so outside links aren't broken.

Reply:
Can you make a code suggestion for this?

Reply:
My GitHub page didn't give me the option to suggest change like I've seen before but it did give me the option to commit